easybuild/easyconfigs/n/numexpr/numexpr-2.2.2-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "PythonPackage"
+
+name = 'numexpr'
+version = '2.2.2'
+
+homepage = 'http://code.google.com/p/numexpr/'
+description = """The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
+ It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
+ code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
+ compiler at runtime."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'usempi': True}
+
+source_urls = [GOOGLECODE_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pythonver = '2.7.10'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+dependencies = [
+    (python, pythonver),
+    ('HDF5', '1.8.13'),
```

Comment:
weird indeed. The other easyconfigs have this too. But I cannot find any reference to HDF5 in the code. So will drop.